src/components/course/EnrollButton.jsx

```diff
@@ -233,7 +233,12 @@ export default function EnrollButton() {
           </>
         );
       }
-      return <DefaultEnrollCta className={classNames(enrollLinkClass, 'disabled')} />;
+      return (
+        <DefaultEnrollCta
+          className={classNames(enrollLinkClass, 'disabled')}
```

Comment:
Should we remove `btn-success` from the `enrollLinkClass` variable since we're passing in `variant="success"` now?

Reply:
Similarly, should we make these same changes in `DeprecatedEnrollButton` since that component is the one still in use when the feature flag to use this `EnrollButton` is not enabled?

Reply:
good catch 👍